docs/core/deploying/native-aot/diagnostics/desktop-platforms.md

```diff
@@ -1,38 +1,24 @@
 ---
-title: Diagnostics and instrumentation
-description: Learn about diagnostics in Native AOT applications
+title: Diagnostics and instrumentation on desktop platforms
+description: Learn about diagnostics in Native AOT applications on desktop platforms
```

Comment:
"desktop platforms" is our internal term with very specific definition. It is not a term that our users understand.

For example, it is very odd to call Linux containers as desktop platform.